charts/clearblade-iot-enterprise/charts/cb-postgres/templates/postgres-cluster.yaml

```diff
@@ -0,0 +1,97 @@
+{{- if .Values.global.dynamicVolumes }}
+apiVersion: postgresql.cnpg.io/v1
+kind: Cluster
+metadata:
+  name: cb-postgres
+  namespace: {{ default "clearblade" .Values.global.namespace }}
+  labels:
+    app: cb-postgres
+  annotations:
+    cnpg.io/skipEmptyWalArchiveCheck: enabled
+spec:
+  # template:
+  #   spec:
+  #     serviceAccountName: {{ .Values.global.gcpGSMServiceAccount }}
+        # tolerations:
+        # - key: node.kubernetes.io/not-ready
+        #   operator: Exists
+        #   effect: NoExecute
+        #   tolerationSeconds: 0
+        # - key: node.kubernetes.io/unreachable
+        #   operator: Exists
+        #   effect: NoExecute
+        #   tolerationSeconds: 0
+  instances: 3
+  bootstrap:
+  {{- if .Values.global.restored }}
+    recovery:
+      source: {{ default "clearblade" .Values.global.folder }}
+  externalClusters:
+    - name: {{ default "clearblade" .Values.global.folder }}
+      barmanObjectStore:
+        destinationPath: "{{ .Values.global.bucket }}"
+        # googleCredentials:
+        #   gkeEnvironment: true
+        googleCredentials:
+          applicationCredentials:
+            name: backup-creds
+            key: gcsCredentials
+        wal:
+          maxParallel: 8
+  {{- else }}
+    initdb:
+      postInitSQL:
+        - CREATE USER cbuser;
+        - ALTER USER cbuser WITH SUPERUSER;
+        - ALTER USER cbuser WITH PASSWORD '{{ .Values.global.pass }}';
+        - CREATE DATABASE admin;
+        - GRANT ALL PRIVILEGES ON DATABASE admin TO cbuser;
+        - CREATE EXTENSION pg_stat_statements;
+        - CREATE EXTENSION pg_trgm;
```

Comment:
We're missing a few commands from the usual `postgres.sh` script we run locally. Do we need any of those?

```
GRANT pg_read_all_stats TO cbuser;
GRANT ALL ON SCHEMA public TO cbuser;
```

Do we also want to enable timescale by default?